Integrations/integration-Cloaken.yml

```diff
@@ -0,0 +1,80 @@
+description: Unshorten urls through a tor proxy server.
+detaileddescription: Cloaken will return the unshortened version of a url;  Cloaken is available for install on AWS marketplace. Visit https://aws.amazon.com/marketplace/ search for Cloaken.  Follow the deployment instructions. 
+configuration:
+- display: ""
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: ""
+  name: server_url
+  defaultvalue: ""
+  type: 0
+  required: true
+script:
+  script: |
+    # The command demisto.command() holds the command sent from the user.
+    from cloakensdk.client import SyncClient
+    from cloakensdk.resources import Url
+
+    def get_client():
+        server = demisto.args(["server"])
+        username = demisto.args()["credentials"]["password"]
+        password  = demisto.args()["credentials"]["identifier"]
+        client = SyncClient(server_url=server,
+                                 username=username,
+                                 password=password)
+        return  client
+
+
+    if demisto.command() == 'test-module':
+        # This is the call made when pressing the integration test button.
+        client = get_client()
+        demisto.results('ok')
+        sys.exit(0)
+
+    if demisto.command() == "cloaken":
+        client = getclient()
+        url = demisto.args()["url"]
+        resource = Url(client)
+        response = resource.unshorten(url)
+        ec={}
+        ec["URL"]={}
+        ec["Cloaken"]=[]
+        context={}
+        if response["status"]=="Success":
+            url_data = response["data"]["unshortened_url"]
+            ec["URL"]["Data"]=url_data
+            context["original_url"]=url
+            context["unshortened_url"]
+            ec["Cloaken"].append(context)
+            demisto.results([{
+                "Contents":context,
+                "ContentsFormat":formats["json"]
+                "EntryType" :entryTypes["note"]
+                "HumanReadable":tableToMarkdown("Cloakened URL: ", context)
+                "EntryContext":ec,
+               }])
+        else:
+            demisto.results([{
+                "Contents":"Error Cloaken Unshorten: " + str(response["data"]),
+                "EntryType":entryTypes["error"],
+                "ContentsFormat":formats["text"]
+                 }]
+  type: python
+  commands:
+  - name: cloaken
+    arguments:
+    - name: url
+      required: true
+      description: url to unshorten
+    description: unshorten a url
+  dockerimage: cypherint/cloakensdk
```

Comment:
Please build the docker image here https://github.com/demisto/dockerfiles

Reply:
TODO(AJ) I will work on this next.

Reply:
I created a pull request for the docker image.  Note I was not able to test the integration with the new docker image yet.  I presume the PR needs to be completed before your process builds and posts it.